sdk/data.go

```diff
@@ -45,13 +45,15 @@ type Event struct {
 
 // EntityData defines all the data related to a particular event from an entity.
 type EntityData struct {
-	Entity    Entity       `json:"entity"`
-	Metrics   []metric.Set `json:"metrics"`
-	Inventory Inventory    `json:"inventory"`
-	Events    []Event      `json:"events"`
+	storer    persist.Storer
```

Comment:
`json:"-"`

Reply:
otherwise it'll try to serialize storer fields https://play.golang.org/p/OBhRgNV_8FC